M6/6.ICE.1/webpack.config.js

```diff
@@ -0,0 +1,11 @@
+// webpack.config.js
+const path = require('path');
+
+module.exports = {
+  mode: "development" , // or "production"
+  entry: "./src/index.js",
+//   output: {
```

Comment:
why is this commented out? can we use the default

```
  output: {
    filename: "main.js",
    path: path.resolve(__dirname, "dist")
  }
```

Reply:
done!